src/test/java/se/sundsvall/supportmanagement/service/mapper/ErrandMapperTest.java

```diff
@@ -59,9 +65,10 @@ void testToErrand() {
 		assertThat(errand.getAssignedUserId()).isEqualTo(ASSIGNED_USER_ID);
 		assertThat(errand.getCategoryTag()).isEqualTo(CATEGORY_TAG);
 		assertThat(errand.getCreated()).isCloseTo(CREATED, within(2, SECONDS));
-		assertThat(errand.getCustomer()).isNotNull();
-		assertThat(errand.getCustomer().getId()).isEqualTo(CUSTOMER_ID);
-		assertThat(errand.getCustomer().getType()).isEqualTo(CustomerType.valueOf(CUSTOMER_TYPE));
+		assertThat(errand.getStakeholders()).isNotNull();
```

Comment:
You can chain hasSize(1) after isNotNull() to save some bytes (or remove null check all together as it will be checked and fail when checking that size is 1).

Reply:
isNotNull() removed